### PR TITLE
Fix translation all words

### PR DESCRIPTION
**Pull request by patel-malav**

### CODE REVIEW

File: html/classhelper.js [hunk 1]
```diff
@@ -830,19 +846,7 @@ class ClassHelper extends HTMLElement {
...
-            footThx.textContent = "X";
-            footerRow.appendChild(footThx);
-        }
-
-        headers.forEach(header => {
-            const th = document.createElement('th');
-            th.textContent = header;
-            footerRow.appendChild(th);
-        });
+        const footerRow = headerRow.cloneNode(true);
```
Comment (rouilj) [inline html/classhelper.js:849]: Good catch.

**Comment by rouilj** [comment]
Except for the doc issue looks good to me.

**Comment by patel-malav** [comment]
@rouilj um doc issues.

you mean should I not have included that section or just refer them to roundup docs for translation?

**Comment by rouilj** [comment]
Including the section is fine Especially since the translation strings can't be automatically extracted by roundup-gettext.

I think your example is wrong. I don't think:
```
msgid "Next"
msgstr "nächste"
```
is going to work. When you set up:
``
const CLASSHELPER_TRANSLATION_KEYWORDS = ["apply", "cancel", "next", "prev", "search", "reset"];
```
you changed "Next" to "next". I think the docs have to change to:
```
msgid "next"
msgstr "nächste"
```

**Comment by patel-malav** [comment]
@rouilj Fixed the docs and added warning for case sensitive msgid 

Merging it into main

**Pull request merged**